src/Language/Fixpoint/Solver/PLE.hs

```diff
@@ -470,20 +470,17 @@ getAutoRws γ ctx =
 evalOne :: Knowledge -> EvalEnv -> ICtx -> Int -> Expr -> IO (EvAccum, FuelCount)
 evalOne γ env ctx i e | i > 0 || null (getAutoRws γ ctx) = do
     ((e', _), st) <- runStateT (eval γ ctx NoRW e) (env { evFuel = icFuel ctx })
-    let evAcc' = if (mytracepp ("evalOne: " ++ showpp e) e') == e then evAccum st else S.insert (e, e') (evAccum st)
+    let evAcc' = if mytracepp ("evalOne: " ++ showpp e) e' == e then evAccum st else S.insert (e, e') (evAccum st)
     return (evAcc', evFuel st)
-evalOne γ env ctx _ e | otherwise = do
+evalOne γ env ctx _ e = do
   env' <- execStateT (evalREST γ ctx rp) (env { evFuel = icFuel ctx })
   return (evAccum env', evFuel env')
   where
     oc :: AbstractOC (OCType Op) Expr IO
     oc = ordConstraints (Mb.fromJust $ restSolver env)
 
     rp = RP oc [(e, PLE)] constraints
-    constraints = foldl go (OC.top oc) []
-      where
-        go c (t, u) = refine oc c t u
-
+    constraints = OC.top oc
```

Comment:
This is probably the most interesting change in this PR.

`foldl go (OC.top oc) []`, which reduces to `OC.top oc`, appears to be a leftover from this change: https://github.com/ucsd-progsys/liquid-fixpoint/commit/78e875e099c59a9d5e9f138bebba2ec157cd3c88#diff-138cb912182d4ea38121c4b70aef046e31f048a879f00bdfa2a03cfd0ac5f5e5L468-R477